components/tyk-pump/README.md

```diff
@@ -61,14 +61,24 @@ You may set `global.redis.addr` and `global.redis.pass` with redis connection st
 
 ### Pump Configurations
 
-| Pump                      | Configuration                                                                                              |
-|---------------------------|------------------------------------------------------------------------------------------------------------| 
-| Prometheus Pump (Default) | Add `prometheus` to `pump.backend`, and add connection details for prometheus under `pump.prometheusPump`. |
-| Mongo Pump                | Add `mongo` to `pump.backend`, and add connection details for mongo under `.global.mongo`.                 |
-| SQL Pump                  | Add `postgres` to `.pump.backend`, and add connection details for postgres under `.global.postgres`.       |
-| Uptime Pump               | Set `pump.uptimePumpBackend` to `'mongo'` or `'postgres'` or `''`                                          |
-| Hybrid Pump               | Add `hybrid` to `.pump.backend`, and setup `.global.remoteControlPlane` section with the required adresses and tokens           |
-| Other Pumps               | Add the required environment variables in `pump.extraEnvs`                                                 |
+| Pump                      | Configuration                                                                                                         |
+|---------------------------|-----------------------------------------------------------------------------------------------------------------------| 
+| Prometheus Pump (Default) | Add `prometheus` to `pump.backend`, and add connection details for prometheus under `pump.prometheusPump`.            |
+| Mongo Pump                | Add `mongo` to `pump.backend`, and add connection details for mongo under `.global.mongo`.                            |
+| Mongo Selective Pump      | Add `mongo-selective` to `pump.backend`, and add connection details for mongo under `.global.mongo`.                  |
+| Mongo Aggregate Pump      | Add `mongo-aggregate` to `pump.backend`, and add connection details for mongo under `.global.mongo`.                  |
+| Postgres Pump             | Add `postgres` to `.pump.backend`, and add connection details for postgres under `.global.postgres`.                  |
+| Postgres Aggregate Pump   | Add `postgres-aggregate` to `.pump.backend`, and add connection details for postgres under `.global.postgres`.        |
+| Uptime Pump               | Set `pump.uptimePumpBackend` to `'mongo'` or `'postgres'` or `''`                                                     |
```

Comment:
what does empty value in `Uptime Pump` mean?

Reply:
@letzya  According to https://github.com/TykTechnologies/tyk-charts/blob/0e29de9f3fad6f2a9d046f0c0f670b77e49d754a/components/tyk-pump/values.yaml#L180-L182, empty value for Uptime Pump means "disabled". So that we do not need an extra field (`enableUptimePump` or sth) to control whether Uptime Pump is enabled or not